system/src/system_power_manager.cpp

```diff
@@ -330,6 +331,10 @@ void PowerManager::loop(void* arg) {
         // Do not re-run DPDM
         self->initDefault(false);
         self->update_ = true;
+      } else if (ev == Event::Wakeup) {
```

Comment:
This event should trigger: `initDefault()` + `handleUpdate()` call here in this thread.